### ISSUE TITLE
COGA TF feedback on NAUR draft - cognitive accessibility considerations still need to be incorporated

### ISSUE THREAD

**Issue by rainbreaw** [w3c/apa#263]
Our main comment is to request that APA does another review of and check against the work that COGA has done in the following documents:

- [Cognitive Accessibility Issue Papers](https://w3c.github.io/coga/issue-papers/)
   - See in particular [7. Voice Systems](https://w3c.github.io/coga/issue-papers/#voice-systems) and [7.2 Challenges for People with Cognitive Disabilities](https://w3c.github.io/coga/issue-papers/#challenges-for-people-with-cognitive-disabilities). This is a mature paper that deals with voicemail-type menus.
- [Conversational Interfaces and Voice Systems](https://docs.google.com/document/d/1B1vCqlU1IF5UmqxhJAy8Khdi-kRQNPalVX8f3lCMr7w/edit#heading=h.9j8rnhzb9sv8)
   - This is a draft issue paper that deals with free-form interfaces such as Alexa.
- [Making Content Usable for People with Cognitive and Learning Disabilities](https://www.w3.org/TR/coga-usable/)
   - All of Making Content Usable’s objectives are relevant to NAUR, but see in particular [Objective 6: Ensure Processes Do Not Rely on Memory](https://www.w3.org/TR/coga-usable/#objective-6-ensure-processes-do-not-rely-on-memory) and [Objective 7: Provide Help and Support](https://www.w3.org/TR/coga-usable/#objective-7-provide-help-and-support-0)

We have summarized more specific feedback in this [COGA TF Summary feedback for NAUR, 7-page Google Doc](https://docs.google.com/document/d/1lVkQv4_lKcuEyane3o2oDtC3Hzu_-YZRJKe8pSk7Qd4/edit?pli=1#), but our summary should not be considered exhaustive or minimize the need to check against the documents referenced above. A plain text version of this document is also available: [COGA TF_ Summary of feedback for NAUR.txt](https://github.com/w3c/apa/files/7696089/COGA.TF_.Summary.of.feedback.for.NAUR.txt).

**Comment by jstanley-mitre** [comment]
The MITRE Chatbot Accessibility Playbook may be helpful here. It maps WCAG and COGA Content Usable to chatbot features. There are three links below, corresponding to three sections in the playbook on content design, interface, and integration of a chatbot into the larger website.

Content: https://mitre.github.io/chatbot-accessibility-playbook/docs/4_2_22.html
Interface: https://mitre.github.io/chatbot-accessibility-playbook/docs/4_3_18.html
Integration: https://mitre.github.io/chatbot-accessibility-playbook/docs/4_4_12.html

**Comment by RealJoshue108** [comment]
Thanks @rainbreaw just FYI we are actively discussing this in Research Questions task force - and I've created a branch with some of the suggested edits

https://raw.githack.com/w3c/apa/naur_coga_edits_2022/naur/index.html

**Comment by RealJoshue108** [comment]
Thanks for the references Jeff - this would useful to discuss on the call next week. I'll suggest to the chairs to  add it to the agenda.

**Comment by RealJoshue108** [comment]
@rainbreaw I think the challenge for us is having stable peer reviewed references for any assertions that are made in our notes. We really can't reference issue papers etc. So the most useful part, from the perspective of having a robust and stable publication, is the refs to content usable.

>All of Making Content Usable’s objectives are relevant to NAUR, but see in particular [Objective 6: Ensure Processes Do Not Rely on Memory](https://www.w3.org/TR/coga-usable/#objective-6-ensure-processes-do-not-rely-on-memory) and [Objective 7: Provide Help and Support](https://www.w3.org/TR/coga-usable/#objective-7-provide-help-and-support-0)

**Comment by jstanley-mitre** [comment]
Here I'll summarize a few items MITRE found in Content Usable that could be relevant and that I don't currently see in the NAUR. I haven't looked at the new branch yet.

- Keep content and workflows brief (Content Usable Design Patterns: 4.4.5, 4.6.3, 4.6.2). I think this could be a new item under NAUR 2.6. If the interface provides lists of options, these should also be brief.
- Do not rely on user's memory (Content Usable Design Pattern 4.7.5; Relevant WCAG criteria 3.3.8 Don't make users enter information twice). This could be a new item under NAUR 2.6.
- Help the user stay safe (Content Usable Design Pattern 4.5.11). Natural language interfaces often ask for personal information and store preferences. How can privacy concerns be addressed?
- When possible, human help should be available (Content Usable Design Pattern 4.8.1), and it should be clear when the automation transfers to a human. The NAUR includes examples with RTT and a sign language interpreter, but this could be generalized for anyone having trouble with the system.
